tests/store/artifact/test_ftp_artifact_repo.py

```diff
@@ -66,6 +68,43 @@ def test_list_artifacts(ftp_mock):
     assert artifacts[1].file_size is None
 
 
+def test_list_artifacts_with_absolute_path(ftp_mock):
+    artifact_root_path = "/experiment_id/run_id/"
+    repo = FTPArtifactRepository("ftp://test_ftp"+artifact_root_path)
+
+    repo.get_ftp_client = MagicMock()
+    call_mock = MagicMock(return_value=ftp_mock)
+    repo.get_ftp_client.return_value = MagicMock(__enter__=call_mock)
+
+    # mocked file structure
+    #  |- file
+    #  |- model
+    #     |- model.pb
+
+    file_path = "/file"
+    file_name = os.path.basename(file_path)
+    dir_path = "/model"
+    dir_name = os.path.basename(dir_path)
+    file_size = 678
+    ftp_mock.cwd = MagicMock(side_effect=[None, ftplib.error_perm, None])
+    ftp_mock.nlst = MagicMock(return_value=[file_path, dir_path])
```

Comment:
```suggestion
    file_path = "file"
    file_name = os.path.basename(file_path)
    dir_path = "model"
    dir_name = os.path.basename(dir_path)
    file_size = 678
    ftp_mock.cwd = MagicMock(side_effect=[None, ftplib.error_perm, None])
    ftp_mock.nlst = MagicMock(return_value=[
       posixpath.join(artifact_root_path, file_path),
       posixpath.join(artifact_root_path, dir_path)
    ])
```

Can we just mock  `ftp_mock.nlst` to return absolute paths?

Reply:
LGTM.